cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
+    }
+})
+
+//get post command
+y.scriptName("connectme")
+y.command({
+    command: 'get-post',
+    describe: 'gets a post',
+    builder: {
+        post: {
+            describe: 'Post to get',
+            demandOption: true,
+            type: 'string'
+        },
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        getpost(argv)
+    }
+})
+//Post command
+y.scriptName("connectme")
+y.command({
+    command: 'post',
+    describe: 'creates a post ',
+    builder: {
+        post: {
+            describe: 'Post name',
+            demandOption: true,
+            type: 'string'
+        },
+        canLike: {
+            describe: 'Possibility to like',
+            demandOption: true,
+            type: 'bool'
+        },
+       canUnlike: {
+           describe: 'Possibility to unlike',
+            demandOption: true,
+            type: 'bool'
+        },
+        likes: {
+            describe: 'Ammount of likes',
+            demandOption: true,
+            type: 'int'
+        },
+        tag: {
+            describe: 'Tag',
+            demandOption: true,
+            type: 'string'
+        },
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        createPost(argv)
+    }
+})
+//update post command
+y.scriptName("connectme")
+y.command({
+    command: 'update-post',
+    describe: 'updates a post',
+    builder: {
+        post: {
+            describe: 'Post name',
+            demandOption: true,
+            type: 'string'
+        },
+        canLike: {
+            describe: 'Possibility to like',
+            demandOption: true,
+            type: 'bool'
+        },
+       canUnlike: {
+           describe: 'Possibility to unlike',
+            demandOption: true,
+            type: 'bool'
+        },
+        likes: {
+            describe: 'Ammount of likes',
+            demandOption: true,
+            type: 'int'
+        },
+        tag: {
+            describe: 'Tag',
+            demandOption: true,
+            type: 'string'
+        },
+
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        updatePost(argv)
+    }
+})
+
+// Delete post
+y.scriptName("connectme")
+y.command({
+    command: 'delete-post',
+    describe: 'deletes a post',
+    builder: {
+        post: {
+            describe: 'Post name',
+            demandOption: true,
+            type: 'string'
+        },
+        canLike: {
+            describe: 'Not possibility to like',
+            demandOption: true,
+            type: 'bool'
+        },
+       canUnlike: {
+           describe: 'Not possibility to unlike',
+            demandOption: true,
+            type: 'bool'
+        },
+        likes: {
+            describe: 'Ammount of likes',
+            demandOption: true,
+            type: 'int'
+        },
+        tag: {
+            describe: 'Tag',
+            demandOption: true,
+            type: 'string'
+        },
+

```

Comment:
why do we need all of this to delete a post? I would expect we only need the postId and nothing more.

Reply:
done